.github/workflows/tests.yml

```
@@ -282,10 +282,13 @@ jobs:
               echo 'PyGObject 3 is available' || echo 'PyGObject 3 is not available'
             )
 
-          python -mpip install --upgrade pyqt5${{ matrix.pyqt5-ver }} &&
-            python -c 'import PyQt5.QtCore' &&
-            echo 'PyQt5 is available' ||
-            echo 'PyQt5 is not available'
+          # PyQt5 does not have any wheels for ARM on Linux.
+          if [[ "${{ matrix.os }}" != 'ubuntu-20.04-arm' ]]; then
+            python -mpip install --upgrade pyqt5${{ matrix.pyqt5-ver }} &&
```

```suggestion
            python -mpip install --upgrade pyqt5${{ matrix.pyqt5-ver }} --only-binary pyqt5 &&
```

The latest CI job seems like it is trying to build pyqt5 from an sdist still.

---

Ah, I messed up the condition, but I've added this as well.